src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -2067,9 +2067,8 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
                             if (!path || path->nodes.empty())
                                 continue;
 
-                            for (auto itr = path->nodes.begin(); itr != path->nodes.end(); ++itr)
+                            for (WaypointNode waypoint : path->nodes)
```

Comment:
should this have & too ?

Reply:
Probably, I'll go though it all again. Just need to set up my devenv again 😊